src/components/allproducts/allproducts.tsx

```diff
@@ -0,0 +1,15 @@
+import ProdPC from "../prodPC/prodPC";
+import ProdPS from "../prodPS/prodPS";
+import ProdXbox from "../prodXbox/prodXbox";
+
+const AllProducts = () => {
+  return (
+    <div>
+      <ProdPC />
```

Comment:
Do we want to have separate components just for products name? Can't we store them in array?